addons/captives/initSettings.sqf

```diff
@@ -31,3 +31,12 @@ private _category = format ["ACE %1", localize LSTRING(DisplayName)];
     false,
     1
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(validBlindfolds), "EDITBOX",
```

Comment:
Vector for security issues here.

Reply:
Yeah, I see how someone with control over forced CBA settings could inject arbitrary SQF code on a client machine...